### PR TITLE
updating released module versions

### PR DESCRIPTION
**Pull request by HerbertYiga**
updating released module versions

### CODE REVIEW

**Comment by dkayiwa** [comment]
What happened to the rest of the module upgrades in the reverted pull request?

**Comment by HerbertYiga** [comment]
> What happened to the rest of the module upgrades in the reverted pull request?

@dkayiwa   well i see all of them are as per the released versions in this pr

**Comment by dkayiwa** [comment]
@HerbertYiga is there any reason why you released the reference application before this pull request got merged?

**Comment by HerbertYiga** [comment]
> @HerbertYiga is there any reason why you released the reference application before this pull request got merged?

i did this before creating this, on my first check things seemed to be fine cc @dkayiwa

**Comment by dkayiwa** [comment]
> well i see all of them are as per the released versions in this pr

Are you sure you are looking at this? https://github.com/openmrs/openmrs-distro-referenceapplication/pull/567/files

**Comment by HerbertYiga** [comment]
> Are you sure you are looking at this? https://github.com/openmrs/openmrs-distro-referenceapplication/pull/567/files

very sure @dkayiwa

**Comment by HerbertYiga** [comment]
could there be some thing you think i can clean up for this? @dkayiwa

**Comment by dkayiwa** [comment]
@HerbertYiga can you look at the above build failure?

**Comment by HerbertYiga** [comment]
@dkayiwa  looking at the logs here https://github.com/openmrs/openmrs-distro-referenceapplication/pull/571/checks?check_run_id=3709862225, i see the ui failing tests dont relate to the changes i made

**Comment by dkayiwa** [comment]
@HerbertYiga are all the changes in this pull request necessary?

**Comment by HerbertYiga** [comment]
> are all the changes in this pull request necessary?

@dkayiwa  i see let me fix that

**Comment by HerbertYiga** [comment]
@dkayiwa hoping we look good now

File: package/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.openmrs.distro</groupId>
 	<artifactId>referenceapplication-package</artifactId>
-	<version>2.12.0-SNAPSHOT</version>
+	<version>2.12.0</version>
```
Comment (dkayiwa) [inline package/pom.xml:7]: Why are you changing the above version?

Comment (HerbertYiga) [reply package/pom.xml]: > Why are you changing the above version?

@dkayiwa fixed this

**Comment by kdaud** [comment]
@dkayiwa the modules updated in this commit has broken with Bamboo build plan and might be the reason why ref app has broken again on release.

**Comment by dkayiwa** [comment]
@HerbertYiga which ci build did you use to release the adminui module?

**Comment by HerbertYiga** [comment]
well i have also noticed that the release of admin ui didnt go as required here https://ci.openmrs.org/browse/AU-AUL-176/log

**Pull request merged**